### ISSUE TITLE
Use email alias in BBFC email where admin user submitted complaint

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/Blocking-Middleware#133]
Where 

1. A complaint is submitted to BBFC: and
2. The originaluser was a Blocked admin user (and may have used a random name)
3. Supply only the Blocked email alias in the body of the BBFC complaint email, eg "Request originally submitted by Edmund Schlater"

**Comment by JimKillock** [comment]
Actually: maybe we should just change all of the emails to the blocked alias, as this is what the ISPs have? (From BBFC's POV, the email and date and network name are there so they can potentially check with the MNO that the website has been blocked and checked by the MNO, prior to them dealing with a complaint).

**Comment by dantheta** [comment]
BBFC escalation form updated, administrator resend function to follow

**Comment by dantheta** [comment]
As far as I can tell, the original reporter's email is not shown in non-BBFC resent reports.

**Comment by JimKillock** [comment]
That's right, we had two classess of alias shown on BBFC reports – "admin users with alias" and "blocked users". If these are amended to  use the blocked alias we are done.
